### ISSUE TITLE
became root

### ISSUE THREAD

**Issue by sunnmas** [anycable/anycable-go#155]
**AnyCable-Go version:1.2
Hello. I use the official docker image, I see that the user is nobody. Is there any option to become root? The certificates that I attach to the container are obtained in another container and belong to root. Accordingly, it is not possible to launch anycable-go --port=443 -ssl_cert=path/to/ssl.cert -ssl_key=path/to/ssl.key due to lack of permissions. Please help. Detailes here: https://qna.habr.com/q/1201372

**Comment by palkan** [comment]
The simplest solution would be to build a custom image to provide custom functionality. You can try multi-stage builds for that, smth like:

```dockerfile
FROM anycable/anycable-go:1 as anycable

FROM scratch

COPY --from=anycable /usr/local/bin/anycable-go  /usr/local/bin/anycable-go
ENTRYPOINT ["/usr/local/bin/anycable-go"]
```

P.S. The alpine suggestion from Habr also looks like a potential solution.

**Comment by sunnmas** [comment]
Thanks for your answer! I have solved the problem of accessing certificates. I generate certificates using the let'sencrypt service. They are in pem format. These are the same certificates that I use for nginx. I have registered entrypoint: /usr/local/bin/anycable-go --port=443 -ssl_cert=/ssl/.../full chain.pem -ssl_key=/ssl.../privkey.pem. The container starts, but gives this in the logs: RC controller initialized: api:50051 (concurrency: 28, enable_tls: false, proto_versions: v1). My browser application gives the following error:  Mixed Content: The page at 'https://...' was loaded over HTTPS, but attempted to connect to the insecure WebSocket endpoint 'ws://.../cable'. This request has been blocked; this endpoint must be available over WSS.
Can I use pem certificates? Or maybe the problem is something else? Everything works on a local computer without https, it doesn't work in production
